frontend/src/components/sign_up/sign_up.css

```diff
@@ -1,130 +1,99 @@
-:root {
-  --input-padding-x: .75rem;
-  --input-padding-y: .75rem;
-}
 
-html,
-body {
-  height: 100%;
-}
+.firstName.col-md.form-control-lg {
```

Comment:
This styling seems to be repeated across the different inputs on this page. Maybe a shared class would be better, then styling specific deviations from that shared style?

Reply:
Done! I actually left the class names pretty generic, as it seemed like we would want this across forms (like no outline appearing onFocus or no weird border shading), but if it causes conflicts, let me know